ports/esp8266/boards/esp8266_common.ld

```diff
@@ -142,7 +142,7 @@ SECTIONS
         *lib/utils/interrupt_char.o*(.literal.mp_hal_set_interrupt_char, .text.mp_hal_set_interrupt_char)
         *drivers/bus/*.o(.literal* .text*)
 
-        build/main.o(.literal* .text*)
+        build-*/main.o(.literal* .text*)
```

Comment:
I think it'll work the same if it's just: `main.o(.literal* .text*)`

Reply:
Fixed.